### ISSUE TITLE
Clarity / comparability of methods

### ISSUE THREAD

**Issue by ElanVB** [ElanVB/indaba-pracs-2019#4]
# Sub issues:

* [x] Consistency to starting x and y of Banana function section #12 
* [x] Update SGD with decay hyper-parameters so that this method is somewhat competitive in the “Putting it all into practice” section #13 
* [x] Fix code variable naming/notation to be consistent (learning rate decay is different to all other sections) #14 
* [x] Pseudocode notation to be consistent (spacing around “*”, variable names, more pseudocode like - less python-like, etc) #15
